frontend/public/components/storage-class-form.tsx

```diff
@@ -891,11 +893,11 @@ export class StorageClassForm_ extends React.Component<
         <>
           {isCheckbox ? (
             <>
-              <div className="checkbox">
+              <div className="checkbox create-storage-class-form__checkbox">
```

Comment:
Need margin-bottom: var(--pf-global--spacer--sm) 0; here to remove the space between checkbox and desc